contracts/simpleKitty/SimpleBridgeKitty.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity 0.4.24;
+
+import "./SimpleKittyCore.sol";
+import "../kitty/Ownable.sol";
+
+contract SimpleBridgeKitty is Ownable, SimpleKittyCore {
+    event Death(uint256 kittyId);
+
+    function mint(
+        uint256 _tokenId,
+        bool _isReady,
+        uint256 _cooldownIndex,
+        uint256 _nextActionAt,
+        uint256 _siringWithId,
+        uint256 _birthTime,
+        uint256 _matronId,
+        uint256 _sireId,
+        uint256 _generation,
+        uint256 _genes,
+        address _owner
+    ) external onlyOwner {
```

Comment:
consider to introduce a role `bridge`. So, `mint` and `burn` will be allowed to be called by an account with the `bridge` role.

Reply:
Please check https://github.com/poanetwork/cryptokitties-xdai-demo/pull/6/commits/4d25a098348f62a453a96ad416766cfb14d7434c